src/backend/recorders/baseRecorder.ts

```diff
@@ -141,7 +160,19 @@ export abstract class BaseRecorder<T extends WebGLObject> implements IRecorder {
     protected abstract delete(instance: T): number;
 
     protected create(functionInformation: IFunctionInformation): void {
-        // Nothing tracked currently on create.
+        // Removes the spector internal calls to leave only the relevant part.
+        const stackTrace = StackTrace.getStackTrace(5, 0);
```

Comment:
The range for this `StackTrace` trimming was not checked. Overall, I wasn't sure why the first entry would have to be removed? Is this only relevant for specific browsers?